template/src/components/PointsOfInterest/PointOfInterest.tsx

```diff
@@ -30,11 +34,16 @@ const PointOfInterest: FC<PointOfInterestProps> = ({ model, onDelete, ...other }
     useWatchAndRerender(model, ["title", "geometry.x", "geometry.y", "color"]);
     const [lon, lat] = xyToLngLat(geometry.x, geometry.y);
     return (
-        <MenuItem {...other}>
-            <div className="PointOfInterest-swatch" style={{ backgroundColor: color.toCss() }} />
+        <MenuItem button={true} {...other}>
```

Comment:
I take this to mean that the hover styling is only applied when `button` is `true`? Does anyone happen to know when this changed and whether it was intentional? It seems weird to me. Vanilla MUI has a hover effect for menu items regardless of whether or not this is set.

Reply:
I think some guy named @eric-parsons maybe changed it? 😛 

https://github.com/geocortex/vertigis-web-sdk/commit/9746c10cd0b009fcb1bada7ad63e266a63658c6d

Reply:
Haha, gotta watch that guy... he's reckless sometimes.

Sorry, I should give a bit more context on what I meant. It was reported to me by our European colleagues that menu items lost their hover effect in their custom code recently. I checked and indeed when you set VIEWER_URL to point to 5.10.0 or earlier, you get a hover effect when you mouse over the items in the POI list, but not in 5.11. Looking into our styling a bit more, it seems that all of our hover styling for list items is defined under the "button" class, and has been this way since forever. Adding the `button` flag fixes it for our sample, but there are two outstanding questions in my mind:
1. How was this working before prior to 5.11.0? Were we always setting `button` to `true` maybe?
2. Should this styling only applied when `button` is set? It makes sense to me not to do this for all list items in general, since a non-interactive `<li>` shouldn't have a hover effect. But a *menu* item is always interactive, otherwise it's not a menu in any meaningful sense. MUI applies hover styling for these regardless of whether or not `button` is set.

Reply:
Hah thanks for the explanation. Was funny to look back at the file's history and see only 1 person committing to it and that person asking who changed things 😛 

`<MenuItem>` shouldn't need `button` set as that's the default anyways and shouldn't change anything. https://github.com/mui-org/material-ui/blob/48bf11343f04db242b69a0a8ceef3c67ac41de4a/packages/material-ui/src/MenuItem/MenuItem.js#L54

You're right, i'm just as confused now. This isn't making sense... Is it styling that Web has added to its theme maybe? Looking through the `react-ui` theme it looks that we apply the hover effects to `ListItem` when `button` is true, which should be the default case when using `MenuItem`.

Reply:
Maybe something with the typing in react-ui or web:

![image](https://user-images.githubusercontent.com/20694976/109828803-d3761c80-7bf1-11eb-9ece-9fbdb70a8358.png)

Reply:
Yeah it is an issue with web adding in a button=undefined that causes the default to be ignored. I will add a PR in web to deal with this.

Reply:
That makes sense. So where are we at with this PR then? Sounds like this would be resolved on the Web side, in which case i don't know that it's worth adding `button` for such a short period of time?

Reply:
OK will remove button for now, and it will get fixed with the next release of web.

Reply:
Turns out it WAS that eric-parsons guy who broke it! 😄